avgsamplerate.go

```diff
@@ -49,9 +56,18 @@ var _ Sampler = (*AvgSampleRate)(nil)
 
 func (a *AvgSampleRate) Start() error {
 	// apply defaults
-	if a.ClearFrequencySec == 0 {
-		a.ClearFrequencySec = 30
+	if a.ClearFrequencyDuration != 0 && a.ClearFrequencySec != 0 {
+		return fmt.Errorf("the ClearFrequencySec configuration value is deprecated; use only ClearFrequencyDuration")
+	}
+
+	if a.ClearFrequencyDuration == 0 && a.ClearFrequencySec == 0 {
+		a.ClearFrequencyDuration = 30 * time.Second
+	} else {
+		if a.ClearFrequencySec != 0 {
```

Comment:
I think this can be an `else if` in all samplers

Reply:
You're right. Thanks.